File: CASHONEWebsiteNET5/wwwroot/model/Order.js
function Order(data) {
    var instance = this;
    instance._datakey = data.Uid;
    instance.Uid = ko.observable(data.Uid);
    instance.OrderTime = ko.observable(data.OrderTime);
    instance.OrderNo = ko.observable(data.OrderNo);
    instance.OrderType = ko.observable(data.OrderType);

    instance.Title = ko.observable(data.Title);
    instance.Title.extend({
        required: true,
        maxLength: 100
    });
    instance.Company = ko.observable(data.Company);
    instance.Company.extend({
        required: true,
        maxLength: 100
    });
    instance.Address = ko.observable(data.Address);
    instance.Address.extend({
        required: true,
        maxLength: 200
    });
    instance.City = ko.observable(data.City);
    instance.City.extend({
        required: true,
        maxLength: 100
    });
    instance.Country = ko.observable(data.Country);
    instance.Country.extend({
        required: true,
        maxLength: 100
    });
    instance.PostalCode = ko.observable(data.PostalCode);
    instance.PostalCode.extend({
        required: true,
        maxLength: 10
    });

    instance.Email = ko.observable(data.Email);
    instance.Email.extend({
        required: true,
        maxLength: 127
    });

    instance.Cell = ko.observable(data.Cell);
    instance.Cell.extend({
        required: true,
        maxLength: 15
    });
    
    instance.Status = ko.observable(data.Status);
    instance.UserId = ko.observable(data.UserId);

    instance.SecretWord = ko.observable(data.SecretWord);
    instance.SecretWord.extend({
        required: true,
        maxLength: 20
    });

    instance.ConfirmSecretWord = ko.observable(data.ConfirmSecretWord);
    instance.ConfirmSecretWord.extend({
        required: true,
        maxLength: 20,
        message: "Secret and confirm secret word are required and must match."
    });

    instance.RowVersion = ko.observable(data.RowVersion); 

    instance.OrderItems = [];
}

File: DataAccessNET5/Models/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

#nullable disable

namespace DataAccessNET5.Models
{
    [Table("Order")]
    [Index(nameof(OrderNo), Name = "IX_Order_OrderNo", IsUnique = true)]
    public partial class Order
    {
        public Order()
        {
            OrderItems = new HashSet<OrderItem>();
        }

        [Key]
        [Column("UID")]
        public Guid Uid { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime OrderTime { get; set; }
        public long OrderNo { get; set; }
        [StringLength(100)]
        public string OrderType { get; set; }
        [StringLength(100)]
        public string Title { get; set; }
        [Required]
        [StringLength(127)]
        public string Email { get; set; }
        [StringLength(15)]
        public string Cell { get; set; }
        [StringLength(100)]
        public string Company { get; set; }
        [Required]
        [StringLength(200)]
        public string Address { get; set; }
        [StringLength(100)]
        public string City { get; set; }
        [StringLength(100)]
        public string Country { get; set; }
        [StringLength(15)]
        public string Status { get; set; }
        [Column("UserID")]
        [StringLength(450)]
        public string UserId { get; set; }
        public byte[] RowVersion { get; set; }
        [StringLength(20)]
        public string SecretWord { get; set; }
        [StringLength(10)]
        public string PostalCode { get; set; }
        [StringLength(1000)]
        public string Remarks { get; set; }

        [InverseProperty(nameof(OrderItem.Order))]
        public virtual ICollection<OrderItem> OrderItems { get; set; }
    }
}
